aws-api-appsync/src/main/java/com/amplifyframework/datastore/appsync/AppSyncExtensions.java

```diff
@@ -129,6 +129,15 @@ public String toString() {
             '}';
     }
 
+    /**
+     * Checks whether the current extension data indicates an authorization related error.
+     * @return True if it is authorization related, false otherwise.
+     */
+    public boolean isAuthorizationError() {
```

Comment:
This is kind of a public facing class, so naming is important.  Perhaps name this method `isUnauthorizedErrorType`?